gdp-src-build/conf/templates/dragonboard-410c.local.conf

```diff
@@ -0,0 +1,5 @@
+include templates/local.inc
+
+MACHINE = "dragonboard-410c"
+
+ACCEPT_EULA_dragonboard-410c = "1"
```

Comment:
Are we sure we want the eula acceptance to be default? Personally I do no take issue to this but we may want to make end users aware that this is the case if left as default. If we were to make it optional, we could handle it with init.sh